e2e/support/test-visualizer-data.ts

```diff
@@ -0,0 +1,557 @@
+const { H } = cy;
+import { SAMPLE_DATABASE } from "e2e/support/cypress_sample_database";
+import type {
+  NativeQuestionDetails,
+  StructuredQuestionDetails,
+} from "e2e/support/helpers/api";
+import { createDataSourceNameRef } from "metabase/visualizer/utils";
```

Comment:
here's your import problem, In general we shouldn't make application code a dependency of tests